Package.swift

```diff
@@ -1,12 +1,20 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.4
 import PackageDescription
 
 let package = Package(
   name: "Lottie",
-  platforms: [.iOS("9.0"), .macOS("10.10"), .tvOS("9.0")],
+  platforms: [.iOS("11.0"), .macOS("10.10"), .tvOS("11.0")],
```

Comment:
I had to bump the minimum platform target from iOS 9 to iOS 11, because the Snapshot Testing library requires iOS 11+. I think this is probably ok, since even iOS 11 is from 2017.

Reply:
This seems fine to me 👍, but we should probably capture this change in release notes and also update podspecs so we have a consistent deployment target